docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md

```diff
@@ -0,0 +1,104 @@
+# Dasharo Compatibility: USB Boot
+
+## Test cases
+
+### UBB001.001 USB detect and boot after coldboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
+operating system after coldboot.
```

Comment:
In the test case desription we should add information, that the test case may be re-done several times to specify the platform and connection stability.